catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/platform/transformer/transformers/miro/MiroIdentifiers.scala

```diff
@@ -0,0 +1,89 @@
+package uk.ac.wellcome.platform.transformer.transformers.miro
+
+import uk.ac.wellcome.models.work.internal.{IdentifierType, SourceIdentifier}
+import uk.ac.wellcome.platform.transformer.source.MiroTransformableData
+
+trait MiroIdentifiers extends MiroTransformableUtils {
+  def getIdentifiers(miroData: MiroTransformableData,
+                     miroId: String): List[SourceIdentifier] = {
+    val miroIDList = List(
+      SourceIdentifier(
+        identifierType = IdentifierType("miro-image-number"),
+        ontologyType = "Work",
+        value = miroId)
+    )
+
+    // Add the Sierra system number from the INNOPAC ID, if it's present.
+    //
+    // We add a b-prefix because everything in Miro is a bibliographic record,
+    // but there are other types in Sierra (e.g. item, holding) with matching
+    // IDs but different prefixes.
+
+    // First, a note: one of the Miro records has a bit of weird data in
```

Comment:
This is the interesting bit – the rest is just copied from the old transformer code.